### ISSUE TITLE
cannot run composer using RUN command

### ISSUE THREAD

**Issue by chaintng** [million12/docker-nginx-php#9]
After build docker image
When it reach RUN composer install command
I got this response

```
Step 6 : RUN composer install --prefer-source --no-interaction
 ---> Running in 06cfef7336f4
/usr/bin/env: php: No such file or directory
The command '/bin/sh -c composer install --prefer-source --no-interaction' returned a non-zero code: 127
```

Please advise.

**Comment by ryzy** [comment]
@chaintan17 thanks for reporting. Which branch are you on? 7.0?

**Comment by chaintng** [comment]
@ryzy yes, i'm on php70 branch :)

**Comment by dimaip** [comment]
@chaintan17 I think I had this issue when using Ansible, and as I had not time to look into it I just used and absolute path to php. I think the problem is that the correct path is being set here doesn't work out for some reason: https://github.com/million12/docker-nginx-php/blob/php70/container-files/config/init/20-php.sh#L13
Don't know when I can look into it properly though, so if you can fix it that would be great!

**Comment by ryzy** [comment]
@chaintan17, @dimaip: did you maybe have a chance to debug it further and find the cause?

**Comment by ryzy** [comment]
Closing, resolved by e4da0da
